lib/fusion_auth/plugs/authorize_jwt.ex

```diff
@@ -98,6 +98,10 @@ defmodule FusionAuth.Plugs.AuthorizeJWT do
         options[:conn_key],
         format(claims, options[:atomize_keys], options[:case_format])
       )
+      |> Plug.Conn.assign(
```

Comment:
Do we need to add any tests to capture this logic?

Reply:
it's already being tested out on the VerusServer side, but we could add some more checks into this repo